x-pack/solutions/security/packages/features/src/security/product_feature_config.ts

```diff
@@ -32,25 +32,57 @@ export const securityDefaultProductFeaturesConfig: DefaultSecurityProductFeature
       },
     },
   },
-  [ProductFeatureSecurityKey.investigationGuide]: {
+
+  [ProductFeatureSecurityKey.externalDetections]: {
     privileges: {
       all: {
-        ui: ['investigation-guide'],
+        ui: ['show', 'external_detections'],
+        api: [APP_ID, 'lists-all', 'lists-read', 'lists-summary', 'rac'],
```

Comment:
The ui/api actions that are added by both `detections` and `externalDetections` productFeatures should remain in the baseKibanaFeature, there's no reason to put them here if they are always present.
I am talking about `show` and `crud` capabilities, and all these api actions.